cpp/ps0001/ps0001_solv.cpp

```diff
@@ -4,12 +4,20 @@ using namespace std;
 
 int main()
 {
-    int n1,n2,n3,sum=0;
-    for (int i =1 ; i <=25 ; i++)
+    // declared 3 variables to store user inputs 0.5/1
+    unsigned short n1,n2,n3,sum=0;
+    
+    for (int i = 1; i <= 25; i++)
     {
+        // prompting user for 3 numbers to add 1/1
         cout << "Enter 3 numbers \n";
-        cin >> n1>>n2>>n3;
-        sum = n1+n2+n3;
-        cout <<"----------"<<"\n"<<sum <<"\n";
+        cin >> n1 >> n2 >> n3;
+
+        // outputting the result of the summation process 2/2
+        sum = n1 + n2 + n3;
+        cout << "----------\n" << sum << "\n";
     }
+
+    return 0;
 }
+// great work 😎 9/10 ⭐

```

Comment:
- [x] program is in the correct path
- [ ] no syntax errors
- [x] no bugs occurs while execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code